src/vs/base/common/platform.ts

```diff
@@ -75,31 +75,8 @@ interface INavigator {
 }
 declare const navigator: INavigator;
 
-// Web environment
-if (typeof navigator === 'object' && !isElectronRenderer) {
```

Comment:
Previous detection expect that Node.js doesn't have `navigator`, now Node.js 21 had, with `navigator.userAgent === 'Node.js/21'` .